guidelines/requests.md

```diff
@@ -3,11 +3,11 @@
 The API accepts only `HTTP POST` requests with `Content-Type` set to `application/json` and JSON content depending on the operation to be performed. All operations follow this address pattern:
 
 ```text
-[ApiAddress]/api/connector/v1/[Resource]/[Operation]
+[PlatformAddress]/api/connector/v1/[Resource]/[Operation]
```

Comment:
Is this correct? We have two urls. The platform and the API. This should stay as API no?

Reply:
yeah, I discussed this with @jirrick when he did the Guidelines rewrite.

Before, when platform & webapp address was the same, all endpoints (in the Operations section) were shown as 
[PlatformAddress]/api/connector/v1/[Resource]/[Operation]

So either we got through every single endpoint and change PlatformAddress to ApiAddress, or keep PlatformAddress for the endpoint URL, and give the webapp url another name (fewer changes needed in the documentation). 

So we went with option 2, and I'm just now realising that we forgot this part of the documentation and it's not aligned with the rest of the documentation. 

![image](https://user-images.githubusercontent.com/57252634/119985887-82dd4800-bfc3-11eb-8c86-e3055c8c13a6.png)

Reply:
Wouldnt it be better to go with ApiAddress for API, WebSocketAddress and AppAddress for the web app? that way, it will nicely match the subdomains (api.mews.com, ws.mews.com, app.mews.com) in future. I dont like platformAddress because it’s too general.